test/commands/log_in_out.test.ts

```diff
@@ -43,4 +49,23 @@ describe('apify login and logout', () => {
 
         expect(isGlobalConfig).to.be.eql(false);
     });
+
+    it('have correctly setup server for interactive login', async () => {
+        await LoginCommand.run(['-m', 'console'], import.meta.url);
+
+        const consoleInfo = spy.mock.calls[0][0];
+        const consoleUrl = /"(http[s]?:\/\/[^"]*)"/.exec(consoleInfo)?.[1];
+
+        const consoleUrlParams = new URL(consoleUrl!).searchParams;
+
+        const localCliPort = consoleUrlParams.get('localCliPort');
+        const localCliToken = consoleUrlParams.get('localCliToken');
+
+        const response = await axios.post(`http://localhost:${localCliPort}/api/v1/login-token?token=${localCliToken}`,
+            { apiToken: TEST_USER_TOKEN },
+            { headers: { 'Content-Type': 'application/json' } },
+        );
+
+        expect(response.status).to.be.eql(200);
```

Comment:
I think here you should also check if the login actually worked, same as in the previous test.